test/components/plots/areaPlotTests.ts

```diff
@@ -129,7 +129,7 @@ describe("Plots", () => {
         var newTwoPointData = [{ foo: 2, bar: 1 }, { foo: 3, bar: 2 }];
         areaPlot.addDataset("newTwo", new Plottable.Dataset(newTwoPointData));
         var allAreas = areaPlot.getAllSelections("newTwo");
-        assert.strictEqual(allAreas.size(), 1, "all areas retrieved");
+        assert.strictEqual(allAreas.size(), 2, "all areas retrieved");
```

Comment:
"line and area retrieved"?

We should check that there is both a `class="line"` and `class="area"` `<path>` in the selection.